glossary.md

```diff
@@ -61,4 +61,4 @@ In revision (or version) control systems, a repository is a data structure which
 
 ## Version control
 
-Version control is the management of changes to source code and the files associated with it. Changes are usually identified by a code, termed the *revision number* (or similar). Each revision is associated with the time it was made and the person making the change thus making it easier to retrace the evolution of the code. Revisions of code can be compared with each other and it makes it easy to restore previous versions.
+Version control is the management of changes to source code and the files associated with it. Changes are usually identified by a code, termed the *revision number* (or similar). Each revision is associated with the time it was made and the person making the change thus making it easier to retrace the evolution of the code. Revisions of code can be compared with each other and to see how content has changed over time.
```

Comment:
```suggestion
Version control is the management of changes to source code and the files associated with it. Changes are usually identified by a code, termed the *revision number* (or similar). Each revision is associated with the time it was made and the person making the change thus making it easier to retrace the evolution of the code. Revisions of code can be compared with each other and makes it easy to see how content has changed over time.
```

Reply:
I think you lost this part when making the change.

Reply:
I think "easy" is not quite right for most audiences; I think the text could say "easier", as a relatively small portion of people describe git as "easy".

Reply:
I buy that, but we still need one more verb in this sentence.

Reply:
This is how I read the sentence: 

```
Revisions of code

  can be compared with each other 
and 
  to see how content has changed over time.
```
Skipping the first clause makes the sentence read as "Revisions of code to see how content has changed over time."

Reply:
I changed my proposal from "makes it easy" to "used" which means that it can be parsed as:

```
Revisions of code can be

  compared with each other 
and 
  used to see how content has changed over time.
```

which makes the sentence read coherently to me.